views/transactions/Transfer.vue

```diff
@@ -53,6 +53,7 @@
           :tokens="availableTokens"
           :balances="availableBalances"
           :max-amount="maxAmount"
+          :approve-required="requireAllowance ?? false"
```

Comment:
Why do we need allowance only for native tokens?

Reply:
Also what happens if there is already enough amount approved? Seems like approve-required would still be set to true?

Reply:
For the input it will still show the lock to indicate that this token requires allowance and yes it will still show if enough amount is approved. We have another component that shows or hides based on if there is enough amount approved. This is the same UX on the deposit view.